.github/SECURITY.md

```diff
@@ -0,0 +1,10 @@
+# Security Policy
+
+## Reporting a Suspected Vulnerability
+
+The PyYAML project encourages responsible disclosure of suspected security
+vulnerabilities. However, we do not offer bug bounties, paid disclosure, or
+paid fixes for discovered vulnerabilities. To report a suspected security
+vulnerability, please e-mail details to `security@pyyaml.org` without creating
```

Comment:
The backticks will make the address not a link. Is that the intention? I can see benefits both ways.